### ISSUE TITLE
"MDN conventions and definitions": broken links to pages that don't exist

### ISSUE THREAD

**Issue by cw118** [mdn/content#16240]
### MDN URL

https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Conventions_definitions

### What specific section or headline is this issue about?

"Archived content" and "Deleted pages"

### What information was incorrect, unhelpful, or incomplete?

The following links are broken:

- `/en-US/docs/Archive` (link text "Archive of obsolete content")
- `/en-US/docs/Archive/B2G_OS` ("B2G (Firefox OS) project")
- `/en-US/docs/Web/API/SharedKeyframeList` (`SharedKeyframeList`)
- `/en-US/docs/Web/API/SharedKeyframeList/SharedKeyframeList` (`SharedKeyframeList()`)

### What did you expect to see?

Working/correct links where applicable (either to the corresponding page or possibly a file/folder on GitHub), or perhaps removing the links if there is no alternative.

### Do you have any supporting links, references, or citations?

I noticed these on localhost because of link flaws.

### Do you have anything more you want to share?

I'd be happy to help fix these myself, but I wanted to check how it should be done first (where we should link instead, etc.) :)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mdn/guidelines/conventions_definitions`
* MDN URL: https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Conventions_definitions
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mdn/guidelines/conventions_definitions/index.md
* Last commit: https://github.com/mdn/content/commit/c8a221f99b57331d0bdf25596cbcbb4ef5f46806
* Document last modified: 2022-05-14T04:03:19.000Z

</details>

**Comment by vagadeeshwar** [comment]
Hey there,

These are links to unwritten pages... you can hover over the links to see the tooltip yourself

They will work once the pages have been written

**Comment by cw118** [comment]
@vagadeeshwar Hello :) I created the issue because I believe the "Archive" links no longer have MDN pages and should/could possibly lead to an archive GitHub repo (e.g. `archived-content`).

For the two `SharedKeyframeList` links, they are for/"lead to" deleted pages, so I wasn't sure if the links should be removed or if they were added to show the styles for deleted/not existing links.

That being said, thank you for taking the time to respond :)
